release-notes/opensearch-release-notes-1.0.0.md

```diff
@@ -0,0 +1,510 @@
+# OpenSearch and Dashboards 1.0.0 Release Notes
+
+## Release Highlights
```

Comment:
I don't see any references to https://github.com/opensearch-project/opensearch or https://github.com/opensearch-project/OpenSearch-Dashboards

Reply:
It seems strange to have all these release notes for the plugins but no notes from these fundamental projects

Reply:
Added links per Kyle Davis suggestions.